### PR TITLE
Added private network param in create server

### PR DESCRIPTION
**Pull request by meklis**

### CODE REVIEW

**Comment by meklis** [comment]
Added private network param in creating a new server

**Comment by LKaemmerling** [comment]
Thank you! Could you please add a changelog line too?

**Comment by meklis** [comment]
Added lines to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.1.1 (12.05.2020)
```
Comment (LKaemmerling) [inline CHANGELOG.md:3]: Should be `## master (XX.XX.XXXX)`

File: src/Models/Servers/Servers.php [hunk 2]
```diff
@@ -114,6 +115,7 @@ public function getById(int $serverId): Server
      * @param string $user_data
      * @param array $volumes
      * @param bool $automount
+     * @param Network[] $networks
```
Comment (LKaemmerling) [inline src/Models/Servers/Servers.php:118]: To be inline with $volumes this should be an `array`  as typehint.

File: src/Models/Servers/Servers.php [hunk 3]
```diff
@@ -126,8 +128,13 @@ public function createInDatacenter(
         $startAfterCreate = true,
         $user_data = '',
         $volumes = [],
-        $automount = false
+        $automount = false,
+        $networks = []
     ): APIResponse {
+        $networkIds = [];
+        foreach ($networks as $network) {
+            $networkIds[] = $network->id;
```
Comment (LKaemmerling) [inline src/Models/Servers/Servers.php:136]: The call should just work with ids like it does with the volume IDs. Not with the whole objects.

File: src/Models/Servers/Servers.php [hunk 4]
```diff
@@ -179,8 +188,13 @@ public function createInLocation(string $name,
                                      $startAfterCreate = true,
                                      $user_data = '',
                                      $volumes = [],
-                                     $automount = false
+                                     $automount = false,
+                                     $networks = []
     ): APIResponse {
+        $networkIds = [];
+        foreach ($networks as $network) {
```
Comment (LKaemmerling) [inline src/Models/Servers/Servers.php:195]: Same as above

**Comment by meklis** [comment]
I don't know why tests is failed

File: src/Models/Servers/Servers.php [hunk 5]
```diff
@@ -126,8 +127,9 @@ public function createInDatacenter(
         $startAfterCreate = true,
         $user_data = '',
         $volumes = [],
-        $automount = false
-    ): APIResponse {
+        $automount = false,
+        $networks = []
+    ): APIResponse {
```
Comment (LKaemmerling) [inline src/Models/Servers/Servers.php:132]:
```suggestion
    ): APIResponse {
```

File: src/Models/Servers/Servers.php [hunk 6]
```diff
@@ -179,8 +183,9 @@ public function createInLocation(string $name,
                                      $startAfterCreate = true,
                                      $user_data = '',
                                      $volumes = [],
-                                     $automount = false
-    ): APIResponse {
+                                     $automount = false,
+                                     $networks = []
+    ): APIResponse {
```
Comment (LKaemmerling) [inline src/Models/Servers/Servers.php:188]:
```suggestion
    ): APIResponse {
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## master (12.05.2020)
```
Comment (LKaemmerling) [inline CHANGELOG.md:3]:
```suggestion
## master (XX.XX.XXXX)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## master (12.05.2020)     
+* Added possibility add private networks in server creating
```
Comment (LKaemmerling) [inline CHANGELOG.md:4]:
```suggestion
* Feature: Allow creation of Servers with Networks
```

**Comment by LKaemmerling** [comment]
Thank you!

**Pull request merged**